tests/epyccel/test_return_stmt.py

```diff
@@ -0,0 +1,68 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring/
+from pyccel.decorators import types
+from pyccel.epyccel import epyccel
+
+def test_single_return_var_assign(language):
+    def single_return_var_assign():
+        y = 3
+        return y
+    epyc_single_return_var_assign = epyccel(single_return_var_assign, language=language)
```

Comment:
This `epyccel` call is the only one which doesn't use `-Werror -Wunused-variable`. Is that intentional?

Reply:
you are right

Reply:
Why doesn't it use them?

Reply:
this test if the return itself not for the temporary variable.

Reply:
Returns are tested in pretty much every other file in epyccel. I'm not sure this is needed in addition to that. It does not test anything that isn't tested later in the file. That said it doesn't do any harm so ok